partials/tabs/coalaonline.html

```diff
@@ -146,8 +146,8 @@ <h4>Optional Settings</h4>
         <div class="row blue-grey darken-4 white-text z-depth-1 no-margin">
           <span>
             <a class="waves-effect grey-text text-lighten-2">
-              <i ng-click="add_bears(section)" class="fa fa-plus-circle bears-icon-small" aria-hidden="true"> &nbsp;ADD BEAR
-              </i>
```

Comment:
Last request for change, doesn't require this change, you can keep this in a separate line.

Reply:
Clarification: I mean `</i>` can be left in a new line.

Reply:
I didn't catch that, means <i></i> do not have to be on the same line. Am I right?